src/Microsoft.Fx.Portability/BreakingChangeParser.cs

```diff
@@ -287,12 +289,12 @@ private static void ParseNonStateChange(BreakingChange currentBreak, ParseState
                     // If a list of allowed categories was provided, make sure that the category found is on the list
                     if (!allowedCategories?.Contains(category, StringComparer.OrdinalIgnoreCase) ?? false)
                     {
-                        throw new InvalidOperationException($"Invalid category detected: {category}");
+                        throw new InvalidOperationException(string.Format(CultureInfo.CurrentCulture, LocalizedStrings.InvalidCategoryDetected, category));
```

Comment:
[Interpolated strings](https://msdn.microsoft.com/en-us/library/dn961160.aspx) uses `string.Format(string, params object[])`, but to use this new feature with CultureInfo, the code needs to be changed to the example below. I wasn't sure which was the easiest approach to follow so I opted for the one above.

```csharp
var formattableString = (System.IFormattable)$"Invalid category detected: {category}"
throw new InvalidOperationException(formattableString.ToString(null, CultureInfo.CurrentCulture))
```